### PR TITLE
[Rebateaccess.com] New ruleset

### PR DESCRIPTION
**Pull request by fuglede**

### CODE REVIEW

File: src/chrome/content/rules/Rebateaccess.com.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+<ruleset name="Rebateaccess.com">
+
+	<target host="americanstandard.rebateaccess.com" />
+	<target host="chat.rebateaccess.com" />
+	<target host="coolermaster.rebateaccess.com" />
+	<target host="hamiltonbeach.rebateaccess.com" />
+	<target host="jabrapartner.rebateaccess.com" />
+	<target host="newegg.rebateaccess.com" />
+	<target host="pantone.rebateaccess.com" />
+	<target host="plantronics.rebateaccess.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Rebateaccess.com.xml]: lepa.rebateaccess.com
enermax.rebateaccess.com
paloalto.rebateaccess.com
zeiss.rebateaccess.com
zeissphoto.rebateaccess.com
manfrotto.rebateaccess.com
olympus.rebateaccess.com
barnesbullets.rebateaccess.com
kaspersky.rebateaccess.com
goldeagle.rebateaccess.com
lasermax.rebateaccess.com
remington.rebateaccess.com
marlinfirearms.rebateaccess.com
icom.rebateaccess.com
bushmaster.rebateaccess.com
para.rebateaccess.com
sportsmanswarehouse.rebateaccess.com
aac.rebateaccess.com
...

So far all of them worked for me.

Comment (fuglede) [inline src/chrome/content/rules/Rebateaccess.com.xml]: Yeah, okay.  I'll add a wildcard rule.

File: src/chrome/content/rules/Rebateaccess.com.xml [hunk 2]
```diff
@@ -11,20 +11,49 @@
...
+	<test url="http://zeissphoto.rebateaccess.com/" />
+	<test url="http://zeiss.rebateaccess.com/" />
+
+	<!-- Exclude lower level domains. -->
+	<exclusion pattern="^http://\w+\.\w+\.rebateaccess\.com/" />
+
+		<test url="http://www.americanstandard.rebateaccess.com/" />
+		<test url="http://www.jabrapartner.rebateaccess.com/" />
+		<test url="http://www.pantone.rebateaccess.com/" />
+		<test url="http://www.plantronics.rebateaccess.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Rebateaccess.com.xml]: exclusion and tests should also include `www.rebateaccess.com`

**Comment by fuglede** [comment]
Thanks, good catch; added.

File: src/chrome/content/rules/Rebateaccess.com.xml (hunk 2 above)
Comment (fuglede) [inline src/chrome/content/rules/Rebateaccess.com.xml]: Thanks, good catch; added.

**Pull request merged**